### DISCUSSION TITLE
Not longer working ...

### DISCUSSION THREAD

**Discussion by MaBo2022** [victornpb/undiscord#445] [1. Questions & Answers]
... after big DiscordUpdate from yesterday, Febr-17, 2023 !

This AddOn is a very good work Developer, but it stops to work, can u fixed please ?

SORRY, my Mistake: it works fine now ! BIG SORRY !!!!

**Comment by DieHardjagged** [comment]
How did you fix it?  
It still doesnt work for me.

**Comment by Smitty620** [comment]
> ... after big DiscordUpdate from yesterday, Febr-17, 2023 !
> 
> This AddOn is a very good work Developer, but it stops to work, can u fixed please ?
> 
> SORRY, my Mistake: it works fine now ! BIG SORRY !!!!

How did you get it to work
